test/scan/scanPerf.chpl

```diff
@@ -0,0 +1,34 @@
+use Time, Memory;
+
+// compute a target problem size if one is not specified
+config const memFraction = 0;
+const totalMem = here.physicalMemory(unit = MemUnits.Bytes);
+const defaultN = if memFraction == 0 then 10
+                                     else (totalMem / numBytes(int)) / memFraction;
+
+config const n = defaultN,
+             printTiming = false,
+             printArray = true;
+
+var A: [1..n] int = 1;
+
+var t: Timer;
+
+// time scan
+t.start();
+var B = + scan A;
+t.stop();
+
+if printTiming then
+  writeln("Scan of ", n, " elements took ", t.elapsed(), " seconds");
+
+if printArray then
+  writeln(B);
+
+// make sure result was correct
+var exp = (n/2 * (1 + n)) + ((n%2) * (n/2 + 1));
+var tot = + reduce B;
+if (tot != exp) then
+  writeln("Verification failed: ", tot, " != ", exp);
+ else
```

Comment:
Not sure if it's worth fixing, but indention looks off

Reply:
Oh for a fix to issue #12327!!!

I don't know if you've picked up on my way of dealing with "probably not important" comments on reviews yet, but if the reviewer spends the time typing in something they think is completely unworthwhile to fix, I still fix it.

Reply:
Yeah, I just don't want to come across as too pedantic so this way it's still "your choice" to fix it :)